### PR TITLE
Resprites Trekkie Uniforms, adds Voyager Uniforms, adds Trekkie Costume Crate

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
I was doing some cleanup downstream and figured I might as well do this one up here. Lord forgive me.
Quick note, didn't touch the Q suit, because I'm actually not a spriter! I'm just making shit up as I go, most of the stuff heavily references other sprites or uses bases/templates, on top of begging my actual spriter friends for advice.

- **Now it's all greyscale! Woa!**
- added Voyager uniforms (They were the sprites I started with downstream, so I carried them along for the ride)
- added a crate with 1 of each (minus the Q suit), with the joke being they're "Scrapped NT Jumpsuit designs"
- removed the stats from the EngSec suit due to them being, like, 10 of 2 types of armor and thats it. It was negligible and I wanted to just toss them to the whole crew as costumes anyways, so that's tossed.
- also made it so that, DESPITE TREK COLORS, Command is gold, EngSec is red, and MedSci is blue. It means it's more uniform and consistent, easier for the code, the eye, and the joke of 'scrapped NT designs'(NT is consistent with their colorcoding)

<details><summary>Sprites in here!</summary>
**(The badge has been fixed on the side-states to not be the 3px but instead be 2px because spin-test. I'm not opening a whole test server again for 1px removed per icon in the same exact place.)**

dreamseeker_udeZX81eqL
dreamseeker_QSojqvJlK1
dreamseeker_0ZBF64WoUD
dreamseeker_D9Z0eStBfj
dreamseeker_v7MuLtfnS2

(GREYSCALE PROOF - All above sprites will be slightly less nice color-wise but are all more uniform)
image

image

</details>

## Why It's Good For The Game
Clean sprites, and actually accessible items.
Plus, someone might find use of them later, in a gateway or a ghost role or something. They're more likely to actually think about it too, because the sprites aren't (as) crusty.

## Changelog

:cl:
add: Added a Trekkie Costume Crate! Now you can wear NT's scrapped jumpsuit designs for only a small cost, all of which will go to NT's emergency anti-copyright savings!
imageadd: Updated the Trekkie uniforms to modern sprite standards, as well as adding Voyager uniforms!
/:cl:

### CODE REVIEW

**Review by Twaticus — CHANGES_REQUESTED** [review]
GAGs please

**Comment by OrionTheFox** [comment]
> GAGs please

gifntext-gif

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -2498,6 +2498,24 @@
...
+					/obj/item/clothing/under/trek/command/voy,
+					/obj/item/clothing/under/trek/command/ent,
+					/obj/item/clothing/under/trek/engsec,
+					/obj/item/clothing/under/trek/engsec/next,
+					/obj/item/clothing/under/trek/engsec/voy,
+					/obj/item/clothing/under/trek/engsec/ent,
+					/obj/item/clothing/under/trek/medsci,
+					/obj/item/clothing/under/trek/medsci/next,
+					/obj/item/clothing/under/trek/medsci/voy,
+					/obj/item/clothing/under/trek/medsci/ent)
```
Comment (Fikou) [inline code/modules/cargo/packs.dm:2516]:
```suggestion
	contains = list(
		/obj/item/clothing/under/trek/command,
		/obj/item/clothing/under/trek/command/next,
		/obj/item/clothing/under/trek/command/voy,
		/obj/item/clothing/under/trek/command/ent,
		/obj/item/clothing/under/trek/engsec,
		/obj/item/clothing/under/trek/engsec/next,
		/obj/item/clothing/under/trek/engsec/voy,
		/obj/item/clothing/under/trek/engsec/ent,
		/obj/item/clothing/under/trek/medsci,
		/obj/item/clothing/under/trek/medsci/next,
		/obj/item/clothing/under/trek/medsci/voy,
		/obj/item/clothing/under/trek/medsci/ent,
	)
```

**Review by Twaticus — APPROVED** [review]
lil confused at how we ended up with more sprites than before but i can look into consolidation after, thanks for setting it up for gags :)

**Comment by OrionTheFox** [comment]
> 

I'm confused as well, I COULD have gotten it to be 1 less icon but it'd have removed some of TNG's base's shading :(

**Comment by OrionTheFox** [comment]
OH, there's more because it's comparing to TG's CURRENT. I added Voyager items, which would be an additional 3, but is instead 2 gags.
It WOULD be less without the Voy base/colors.

File: code/datums/greyscale/config_types/greyscale_configs.dm [hunk 2]
```diff
@@ -870,4 +880,3 @@
 	name = "Security Encryptionkey"
 	icon_file = 'icons/obj/radio.dmi'
 	json_config = 'code/datums/greyscale/json_configs/encryptionkey_security.json'
```
Comment (OrionTheFox) [inline code/datums/greyscale/config_types/greyscale_configs.dm:882]:
```suggestion
	json_config = 'code/datums/greyscale/json_configs/encryptionkey_security.json'

```
Whoops lets not delete the End of File

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -2506,6 +2506,26 @@
 		var/item = pick_n_take(L)
 		new item(C)
 
+/datum/supply_pack/costumes_toys/trekkie
+	name = "Trekkie Costume Crate"
+	desc = "Wear the scrapped concepts for NanoTrasen's jumpsuits, based off popular late-20th century Earth media! While they couldn't be used for the official jumpsuits due to copyright infringement, it's been assured that they can still legally be sold under the label of being 'failed designs'."
```
Comment (Fikou) [inline code/modules/cargo/packs.dm:2511]: it's not NanoTrasen

**Review by Fikou — CHANGES_REQUESTED** [review]

File: code/modules/cargo/packs.dm (hunk 3 above)
Comment (OrionTheFox) [reply code/modules/cargo/packs.dm:2511]: ? all the paperwork has it as NanoTrasen, as well as all the traitor posters

Comment (OrionTheFox) [inline code/modules/cargo/packs.dm:2511]:
```suggestion
	desc = "Wear the scrapped concepts for Nanotrasen's jumpsuits, based off popular late-20th century Earth media! While they couldn't be used for the official jumpsuits due to copyright infringement, it's been assured that they can still legally be sold under the label of being 'failed designs'."
```
Some day I'll not mess this up.

**Comment by Farquaar** [comment]
Voyager is underrated
I stand by this

**Comment by CoffeeDragon16** [comment]
SoMuchWin

**Comment by Imaginos16** [comment]
Seems fine I think or something idk

**Pull request merged**